projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -783,7 +784,7 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
     if (group === 'left') {
       translateXY(styles, offsetX, 0);
     } else if (group === 'right') {
-      const bodyWidth = parseInt(this.innerWidth + '', 0);
+      const bodyWidth = parseInt(this.innerWidth + '', 10);
```

Comment:
`this.innerWidth` is defined as number, so the `parseInt()` isn't necessary

Reply:
You are right as always! I found also another location for it. Updated.